development/kops/install_requirements.sh

```diff
@@ -31,18 +31,7 @@ fi
 if [ ! -x ${KOPS} ]
 then
     echo "Determine kops version"
-    if [ "${RELEASE_BRANCH}" == "1-18" ]
-    then
-        KOPS_VERSION="v1.18.3"
-    else
-        if [ "${RELEASE_BRANCH}" == "1-20" ]
-        then
-            KOPS_VERSION="v1.20.0-beta.2"
-        else
-            KOPS_VERSION="v1.19.0-beta.3"
-        fi
-    fi
-
+    KOPS_VERSION="v1.20.0-beta.2"
```

Comment:
I was thinking someday it might be nice to have a KOPS_VERSION file like we have GIT_TAG maybe.